### PR TITLE
Adding NewRedisService

### PR DESCRIPTION
**Pull request by chug2k**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -40,3 +42,26 @@ func (s *Service) NewFirebaseClient() (firebase.Client, error) {
 	}
 	return firebase.NewClient(baseUrl, authToken, nil), nil
 }
+
+
+func (s *Service) NewRedisClient() (redis.Conn, *redis.PubSubConn, error) {
+	redisTcpAddr, err := s.Env.GetString("REDIS_PORT_6379_TCP_ADDR")
```
Comment (JustinTulloss) [inline client.go:48]: You're not doing anything with these errors. They should be `_` or returned if they occur.

**Comment by JustinTulloss** [comment]
Hmm, I feel like the `NewRedisClient` method that's appropriate for this just looks up the environment variables and calls `Dial`. The two connections for publishing and subscribing is pretty specific to what we're doing with it, that should probably be elsewhere (but call this in order to get the underlying connection)

File: client.go (hunk 1 above)
Comment (chug2k) [inline client.go:48]: i did see that and was wondering what you would say

File: client.go [hunk 2]
```diff
@@ -40,3 +42,32 @@ func (s *Service) NewFirebaseClient() (firebase.Client, error) {
...
+	if err != nil {
```
Comment (chug2k) [inline client.go]: good?

**Comment by JustinTulloss** [comment]
:shipit:

**Pull request merged**